PEERS

```diff
@@ -7,33 +7,41 @@
 # Note: this list may be incorrect, may change, or become out of date at any
 # time without notice.
 
-# web3-storage-sv15
-/ip4/139.178.69.155/tcp/4001/p2p/12D3KooWR19qPPiZH4khepNjS3CLXiB7AbrbAD4ZcDjN1UjGUNE1
-# web3-storage-sv15-2
-/ip4/139.178.68.91/tcp/4001/p2p/12D3KooWEDMw7oRqQkdCJbyeqS5mUmWGwTp8JJ2tjCzTkHboF6wK
-# web3-storage-sv15-3
-/ip4/139.178.69.93/tcp/4001/p2p/12D3KooWRi18oHN1j8McxS9RMnuibcTwxu6VCTYHyLNH2R14qhTy
-# web3-storage-sv15-4
-/ip4/139.178.91.227/tcp/4001/p2p/12D3KooWKhPb9tSnCqBswVfC5EPE7iSTXhbF4Ywwz2MKg5UCagbr
-# web3-storage-sv15-5
-/ip4/139.178.91.231/tcp/4001/p2p/12D3KooWAdxvJCV5KXZ6zveTJmnYGrSzAKuLUKZYkZssLk7UKv4i
 # web3-storage-am6
 /ip4/147.75.33.191/tcp/4001/p2p/12D3KooWPySxxWQjBgX9Jp6uAHQfVmdq8HG1gVvS1fRawHNSrmqW
```

Comment:
```suggestion
```

Reply:
This was removed right?

Reply:
Forget, I see not

Reply:
Some of the machines had cluster disabled, but still have an IPFS node running.